index.ts

```diff
@@ -0,0 +1,217 @@
+import * as adapter from "@api3/airnode-adapter";
+import * as node from "@api3/airnode-node";
+import { AirnodeRrpFactory } from "@api3/airnode-protocol";
+import * as dotenv from "dotenv";
+import * as ethers from "ethers";
+import * as fs from "fs";
+import { each, isEmpty } from "lodash";
+import * as path from "path";
+import { safeDecode } from "./node/abi-encoding";
+//TODO: remove and use @api3/airnode-node import
+import { buildEVMProvider } from "./node/evm-provider";
+//TODO: remove and use @api3/airnode-node import
+import { removeKey, removeKeys } from "./node/object-utils";
+//TODO: remove and use @api3/airnode-node import
+import { getReservedParameters, RESERVED_PARAMETERS } from "./node/parameters";
+//TODO: remove and use @api3/airnode-node import
+import { deriveSponsorWallet } from "./node/wallet";
+//TODO: remove and use "@api3/airnode-protocol" import;
+import RrpBeaconServer from "./RrpBeaconServer.json";
+
+export const handler = async (event: any = {}): Promise<any> => {
+  // **************************************************************************
+  // 1. Load config (this file must be the same as the one used by the node)
+  // **************************************************************************
+  const secretsPath = path.resolve(`${__dirname}/config/secrets.env`);
+  const secrets = dotenv.parse(fs.readFileSync(secretsPath));
+
+  const configPath = path.resolve(`${__dirname}/config/config.json`);
+  const config = node.config.parseConfig(configPath, secrets);
+
+  const { chains, nodeSettings, triggers, ois, apiCredentials } = config;
+  if (isEmpty(chains)) {
+    throw new Error(
+      "One or more chains must be defined in the provided config"
+    );
+  }
+  chains
+    .filter((chain: node.ChainConfig) => chain.type === "evm")
+    .forEach(async (chain: node.ChainConfig) => {
+      each(chain.providers, async (_, providerName) => {
+        // **************************************************************************
+        // 2. Init provider, AirnodeRrp, RrpBeaconServer and Airnode wallet
+        // **************************************************************************
+        const chainProviderUrl = chain.providers[providerName].url || "";
+        const provider = buildEVMProvider(chainProviderUrl, chain.id);
+
+        const airnodeRrp = AirnodeRrpFactory.connect(
+          chain.contracts.AirnodeRrp,
+          provider
+        );
+
+        // TODO: use factory class to create contract instead
+        //   const rrpBeaconServer = RrpBeaconServerFactory.connect(RrpBeaconServer.address, provider);
+        const abi = RrpBeaconServer.abi;
+        const rrpBeaconServer = new ethers.Contract(
+          chain.contracts.RrpBeaconServer,
```

Comment:
That's cool, I was wondering where we would put this

Reply:
I'm thinking that this config change should be done in airnode package with a validator update since Beacon Server  probably will not only be used by Airkeeper.

Reply:
If we're going to use a separate config file for airkeeper that complements config.json (which I think we had decided at the call yesterday) then we can put this field in that field instead of the actual config.json